examples/react-example/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useMatomo } from '@datapunt/matomo-tracker-react'
+
+const App = () => {
+  const { trackPageView, trackEvent } = useMatomo()
+
+  // Track page view
+  React.useEffect(() => {
+    trackPageView({
+      documentTitle: 'Page title', // optional
+      customDimensions: [
+        {
+          id: 1,
+          value: 'loggedIn',
+        },
+      ], // optional
+    })
+  }, [trackPageView])
```

Comment:
I don't think we need to specify this dependency as we only want to call this method once we might as well use an empty list.